### ISSUE TITLE
Either the installation dosc or build scripts need updating

### ISSUE THREAD

**Issue by palisadoes** [exogen/graphql-markdown#97]
1. If you follow the installation documentation, the build scripts create an `index.js` file in the `src/` directory which uses the parameters required by the  `graphql-markdown` script.
2. The documentation refers to a `graphql-markdown` script, but does not mention the location.

The build scripts should create an output executable that matches the documentation, and/or the documentation should be updated.

It looks like a relatively easy thing to fix.

**Comment by exogen** [comment]
Good idea, updated in 67083db! There's a standard location where npm links scripts, but it's not obvious if you haven't played with executable packages before.
